examples/tomo/ray_trafo_parallel_3d.py

```diff
@@ -16,10 +16,10 @@
 detector_partition = odl.uniform_partition([-30, -30], [30, 30], [558, 558])
 # Discrete reconstruction space
 
-# Astra cannot handle axis aligned origin_to_det unless it is aligned
-# with the third coordinate axis. See issue #18 at ASTRA's github.
-# This is fixed in new versions of astra, with older versions, this could
-# give a zero result.
+# Some versions of ASTRA cannot handle axis-aligned origin_to_det unless
+# it is aligned with the third coordinate axis, see issue #18 at ASTRA's
+# GitHub. This is fixed in ASTRA v1.7.2.; with older versions, the following
+# may produce a zero result.
```

Comment:
What exactly was the issue here? Did this occur only in parallel beam 3d? And does axis-aligned mean the case where the rotation axis was the same as the vector from origin to detector at start?
Wouldn't that be a silly setup?